linter.py

```diff
@@ -85,20 +69,8 @@ def cmd(self):
 
         command.append('--json')  # need this for simpler error handling
 
-        c = self.build_cmd(command)
-        persist.debug('flow attempting to run from: {}'.format(c))
         return c
 
-    def context_sensitive_executable_path(self, cmd):
-        """
-        Return the user-configured path to flow, if any.
-
-        Otherwise falls back to NodeLinter for finding the executable.
-        """
-        path = self.get_view_settings()['flow-bin']
-        return (False, path) if path is not None \
```

Comment:
If we remove flow-bin, we don't need the override here. Otherwise you must return (True, path) for success